modules/candidate_parameters/ajax/getData.php

```diff
@@ -51,11 +51,12 @@
 case 'candidateDOB':
     echo json_encode(getDOBFields());
     exit;
+case 'candidateDOD':
+    echo json_encode(getDODFields());
+    exit;
 default:
     header("HTTP/1.1 404 Not Found");
-    exit;
```

Comment:
Why is this exit removed?

Reply:
@driusan I believe a rebase mistake. There was an earlier one that caused a bug.

Reply:
oops, yup it was a rebase mistake. 
I'll issue another PR to add it back

Reply:
you should probably re-rebase it before you make the fix..